src/components/calc.js

```diff
@@ -0,0 +1,51 @@
+// import React from 'react';
+// import './calc.css';
+
+// const Calculator = () => (
+//   <div className="calculator">
+//     <div className="calculator-display">
+//       <input type="text" value="0" readOnly />
+//     </div>
+//     <div className="calculator-buttons">
+//       {/* Your calculator buttons go here */}
+//     </div>
+//   </div>
+// );
+
+// export default Calculator;
+
```

Comment:
- [ ] It is a best coding practice to not have unused commented code in our codebase as they make our code look dirty. Kindly remove all commented unused codes from this file 👍🏽

Reply:
Done